cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -179,9 +179,9 @@ func WriteKubeletConfigFiles(cfg *kubeadmapi.InitConfiguration, patchesDir strin
 }
 
 // GetKubeletDir gets the kubelet directory based on whether the user is dry-running this command or not.
-func GetKubeletDir(dryRun bool) (string, error) {
+func GetKubeletDir(dryRun bool, kubeletConfigDir string) (string, error) {
```

Comment:
this function is not used anywhere else as far as i can tell,

just unroll its contents in  WriteKubeletConfigFiles
it makes  WriteKubeletConfigFiles less confusing.